src/pages/home/report/ReportActionsView.js

```diff
@@ -137,6 +139,8 @@ class ReportActionsView extends React.Component {
         setNewMarkerPosition(this.props.reportID, oldestUnreadSequenceNumber);
 
         fetchActions(this.props.reportID);
+
+        markEnd('ReportActionsView Mounting');
```

Comment:
Similar information is already captured as the component is wrapped with `withRenderTrace` 
But I left that so you can see a comparison of what the <Profiler> captures and what manually inserted marks capture

![image](https://user-images.githubusercontent.com/12156624/130153716-d191ff8d-aa91-4526-8bc0-cf709914479f.png)

Mounting take longer to complete as it seems it's not called immediately after render
The "bubbles" are re-renders triggered by child component (ReportActionItems) rendering

Reply:
Interesting. Are we planning to leave this in? I haven't hooked this up yet. But if `withRenderTrace` mostly achieves the same thing I'd say it's fine to just keep that and lose this other metric unless we want to time it and log in prod/release builds.

Reply:
I don't think we should keep it, and I prefer the profiler as it will show all the renders not just the mount. 
Just wanted to note that `mount` is initial `render` + a bit more time